### DISCUSSION TITLE
Alpine.js bundled with Livewire 3 not working with webpack

### DISCUSSION THREAD

**Discussion by gjm** [livewire/livewire#7006] [7. Bugs (archived)]
### Livewire version

3.0.5

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

I'm (still) using Mix on Laravel 10. I followed the install instruction to manually bundle Livewire and Alpine and manually include assets in my layout:

```
<head>
     ...
     @livewireStyles
    <link href="{{ asset(mix('/css/app.css')) }}" rel="stylesheet" type="text/css">
</head>
<body>
    ...
    @livewireScriptConfig
    <script src="{{ asset(mix('/js/app.js')) }}"></script>
</body>
```

The bundle builds successfully but when I try to load a Livewire component I get this error:

`Alpine Expression Error: func(...).catch is not a function`

The call stack:

```
Uncaught TypeError: func(...).catch is not a function
    at livewire.esm.js:1858:25
    at tryCatch (livewire.esm.js:1782:24)
    at Object.evaluate (livewire.esm.js:1806:7)
    at livewire.esm.js:8115:35
    at callAndClearComponentDebounces (livewire.esm.js:8100:3)
    at Proxy.<anonymous> (livewire.esm.js:8114:7)
    at livewire.esm.js:1826:27
    at tryCatch (livewire.esm.js:1782:24)
    at livewire.esm.js:3658:9
    at handler4 (livewire.esm.js:3030:29)
```

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by manshu** [comment]
I am.having the same issue 
![image](https://github.com/user-attachments/assets/722b8bb1-5e6b-46ad-b38f-abee3972cc67)
![image](https://github.com/user-attachments/assets/316959f7-efef-4203-af56-1f88553b4622)

**Comment by Yinci** [comment]
Pretty annoying that webpack, although no longer maintained but still supported by Laravel 11, is seemingly unusable for Livewire 3, because of this issue, and Livewire 2 isn't supported for Laravel 11, meaning to be able to upgrade to Laravel 11 you NEED to upgrade to Livewire 3 AND Vite, meaning you need to do one massive upgrade in one go. It's also baffling a year has passed and there's still no official workaround to make it work with webpack...

**Reply by manshu** [reply]
And that’s why, i’ve left the ecosystem.

**Reply by Yinci** [reply]
@manshu I mean, fair enough, but what's the alternative? So far with all other options I've tried only Livewire has been the most flawless design flow and easiest to debug. Other option would include crafting a personal bridge between front and backend but that seems like a waste of time with so many options out there.

**Reply by manshu** [reply]
I moved to MERN stack.

**Reply by lrljoe** [reply]
You shouldn't be using something that is no longer maintained ultimately.   

Reducing the performance gains/security for everyone else, because you don't want to update, doesn't seem fair.
